.github/workflows/testing.yml

```diff
@@ -41,8 +45,13 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake -DCMAKE_BUILD_TYPE=${{ matrix.mode }} -DTNTCXX_BUILD_TESTING=ON -DTNTCXX_ENABLE_SSL=OFF .. 
+          cmake -DCMAKE_BUILD_TYPE=${{ matrix.mode }} -DTNTCXX_BUILD_TESTING=ON -DTNTCXX_ENABLE_SSL=ON ..
           make -j
 
+      - name: test without SSL
+        if: matrix.test-ssl == 'Disabled'
+        run: cd build && ctest --output-on-failure -E ClientSSL.test
```

Comment:
[feel free to ignore, just for your knowledge]

Nit: you could use the `${{ matrix.test-ssl == 'Disabled' && 'ClientSSL.test' || ''}}` syntax (see example [here](https://docs.github.com/en/actions/learn-github-actions/expressions#example)) just like in Lua to simplify the code.